src/components/composer/formula_assistant/formula_assistant.xml

```diff
@@ -1,6 +1,12 @@
 <templates>
   <t t-name="o-spreadsheet-FunctionDescriptionProvider">
-    <div class="o-formula-assistant-container user-select-none shadow">
+    <div
+      class="o-formula-assistant-container shadow"
+      tabindex="0"
+      t-on-pointerdown=""
```

Comment:
hovering a cell in the grid puts the focus back on the composer, even if some text of the formula assistant was selected. I'm guessing anything that triggers a render does that ? 

we probably don't care ?

Reply:
I got a shitty fix for that specific situation, i'll let you judge

Reply:
I'm leaning towards "not worth the ugly fix"